nixos/doc/manual/release-notes/rl-2305.section.md

```diff
@@ -77,7 +77,7 @@ In addition to numerous new and updated packages, this release has the following
 
 - [frigate](https://frigate.video), an open source NVR built around real-time AI object detection. Available as [services.frigate](#opt-services.frigate.enable).
 
-- [fzf](https://github.com/junegunn/fzf), a command line fuzzyfinder. Available as [programs.fzf](#opt-programs.fzf.fuzzyCompletion).
+- [fzf](https://github.com/junegunn/fzf), a command line fuzzyfinder. Available as [programs.fzf](#opt-programs.fzf.enable).
```

Comment:
Not sure if that's ok but the manual seems to build against the current state not the state it was made so I get a build-error of the changelog if the old option is still referenced with #opt here. Alternative would be to remove the link "#opt" and leave it as text.